tests/Resources/Page/OCPDashboard/Builds/Builds.robot

```diff
@@ -77,3 +77,27 @@ Wait Until Build Status Is
     [Arguments]    ${namespace}    ${build_name}    ${expected_status}=Complete    ${timeout}=20 min
     Wait Until Keyword Succeeds    ${timeout}    1 min
     ...    Build Status Should Be    ${namespace}    ${build_name}    ${expected_status}
+
+Provoke Image Build Failure
+    [Documentation]    Starts New Build after some time it fail the build and return name of failed build
+    [Arguments]    ${namespace}    ${build_name_includes}    ${build_config_name}    ${container_to_kill}
+    ${build} =    Search Last Build    namespace=${namespace}    build_name_includes=${build_name_includes}
+    Delete Build    namespace=${namespace}    build_name=${build}
+    ${failed_build_name} =    Start New Build    namespace=${namespace}
+    ...    buildconfig=${build_config_name}
+    ${pod_name} =    Find First Pod By Name    namespace=${namespace}    pod_start_with=${failed_build_name}
+    Wait Until Build Status Is    namespace=${namespace}    build_name=${failed_build_name}
+    ...    expected_status=Running
+    Wait Until Container Exist  namespace=${namespace}  pod_name=${pod_name}  container_to_check=${container_to_kill}
```

Comment:
Please, add a Sleep 60s because killing the pod sometimes fail because it's not running yet
```
Wait Until Container Exist  namespace=${namespace}  pod_name=${pod_name}  container_to_check=${container_to_kill}
Sleep    60s    reason=Waiting extra time to make sure the container has started
```

Reply:
Updated.